nautobot/ipam/templates/ipam/rir_retrieve.html

```diff
@@ -0,0 +1 @@
+{% extends 'generic/object_retrieve.html' %}
```

Comment:
Is this needed at all @glennmatthews @HanlinMiao ?

Reply:
Shouldn't be needed after #6851 was done.